Laravel.gitignore

```diff
@@ -21,3 +21,5 @@ Homestead.yaml
 Homestead.json
 /.vagrant
 .phpunit.result.cache
+/.idea
+/.vscode
```

Comment:
.vscode folder has container useful configurations to start with "clean cloning".

See http://go.microsoft.com/fwlink/?LinkId=827846